src/styles/List.scss

```diff
@@ -0,0 +1,42 @@
+/* Include the padding and border in an element's total width and height */
+* {
+    box-sizing: border-box;
+  }
+  /* Style the close button */
+  .close {
+    position: absolute;
+    right: 0;
+    top: 0;
+    padding: 12px 16px 12px 16px;
```

Comment:
Why did you use this sentence?
```css
padding: 12px 16px;
```

It's more clear and better right?

Reply:
The sentence has been modified to the indicated.